src/ballet/ed25519/avx512/fd_r43x6_inl.h

```diff
@@ -152,12 +152,18 @@ FD_PROTOTYPES_BEGIN
 /* FD_R43X6_QUAD_LANE_SUB_FAST does:
      D = [ (imm0 ? (PX-QX) : SX) (imm1 ? (PY-QY) : SY) (imm2 ? (PZ-QZ) : SZ) (imm3 ? (PT-QT) : ST) ]
    imm* should be in [0,1]. */
-
 #define FD_R43X6_QUAD_LANE_SUB_FAST( D, S, imm0,imm1,imm2,imm3, P, Q ) do { \
```

Comment:
My guess here is this modification is subtracting Q from a "big zero" such that reasonable range limbs, the subtraction doesn't wrap and the add of the resulting "-Q" to "P" results in reasonable range output limbs.  If this is correct, I'm a little bit worried about this as it:

- It doesn't look like this would yield the exact same range limb result as the scalar fd_r43x6_sub_fast.  These quad operations are meant to be identical to their scalar counterparts to make developers and numerical analysis life saner.  (E.g. do all numerical analysis in scalar and turn the crank to group into quads and then turn the crank to convert to fast quads without any new numerical analysis).

- Takes slight more operations per lane than fd_r43x6_sub_fast.

Suggest either:

- Keeping the FD_R43X6_QUAD_LANE_SUB_FAST / fd_r43x6_sub_fast as is and add a FD_R43X6_QUAD_LANE_SUB_ALT / fd_r43x6_quad_lane_sub_alt pair with some documentation about the range limbs expected on entry or exist.

- Update fd_r43x6_sub_fast to match with corresponding documentation changes.

I suspect the former is actually less work (e.g. don't have to go through all the places sub_fast is used and update documentation and analysis around existing usages of sub_fast / QUAD_LANE_SUB_FAST.  But YMMV.

Reply:
I fixed so that FD_R43X6_QUAD_LANE_SUB_FAST / fd_r43x6_sub_fast match.
I'll do another pass to validate performance, and if I see anything weird I'll do the _alt versions instead.